### PR TITLE
Replace descriptions of MNIST with ones of FashionMNIST

### LINKED ISSUE #71: Replace outdated docstring in examples using FashionMNIST dataset
## Motivation

Due to the instability of MNIST download, this repo's example codes have used FashionMNIST instead of MNIST.
However, some docstring is supposed to be mentioning MNIST, `hand-written digit recognition`, even though they use FashionMNIST as follows:

> In this example, we optimize the validation accuracy of hand-written digit recognition using
> Catalyst, and FashionMNIST. We optimize the neural network architecture.

The lines come from https://github.com/optuna/optuna-examples/blob/6a6b20ad634627eebb3e7e104f73b70b45c6e624/pytorch/catalyst_simple.py.

## Description
Revise the docstring for FashionMNIST not MNIST.

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

Resolve #71

### CODE REVIEW

**Review by toshihikoyanase — APPROVED** [review]
I'm sorry for the delayed response. Looks perfect. Thank you!
